operations/helm/charts/mimir-distributed/templates/distributor/distributor-svc-headless.yaml

```diff
@@ -15,6 +15,10 @@ spec:
   type: ClusterIP
   clusterIP: None
   ports:
+    - port: {{ include "mimir.serverHttpListenPort" .}}
```

Comment:
The one concern I have is whether metamonitoring will start scraping this automatically, resulting in duplicated distributor metrics. Needs a test - currently manual.

Reply:
The question is fair.
I have one servicemonitor for the entire mimir instance, I did not notice any duplication in the metrics. I will study more carefully.

Reply:
And you are not worried about a possible similar problem for other mimir components, such as query-frontend. There is also an http port in the service and in the headless service

Reply:
we have the `prometheus.io/service-monitor: "false"` annotation on all headless services, this shouldn't be a problem IIRC

Reply:
@krajorama @dimitarvdimitrov In that case, there are no more problems to accept the PR?

Reply:
I think not, please update the changelog with the suggestion and for latest main. And we can merge.